docs/source/delta-sharing.md

```diff
@@ -0,0 +1,204 @@
+ 
+---
+description: Learn how to perform reads on Delta Sharing tables.
+---
+
+# Read Delta Sharing Tables
+
+Delta Sharing supports most of the options provided by <AS> DataFrame read for performing batch/streaming/cdf reads on tables.
+Delta Sharing doesn't support writing to a shared table. Please refer to the [delta sharing repo](https://github.com/delta-io/delta-sharing/blob/main/README.md) for more details. 
+
+For Delta Sharing reads on shared tables with advanced delta features such as DeletionVectors and ColumnMapping, 
```

Comment:
May be make a table and list down what are the read table features that are supported and the version?

| Read Table Feature | Available since <Delta> version|
| Deletion Vectors (link to the DV page) | 3.1.0|

Reply:
added